pl_examples/basic_examples/async_gpu_template.py

```diff
@@ -0,0 +1,95 @@
+"""
+Runs a model on a single node across multiple gpus.
```

Comment:
Pls add some a bit longer description and a note changelog

Reply:
Done!